.eslintrc.ui.js

```diff
@@ -24,10 +24,11 @@ module.exports = {
         varsIgnorePattern: '(^_$|^h$)',
       },
     ],
+    strict: 'off',
   },
-  parser: 'babel-eslint',
```

Comment:
this and the 2020 are in the root config.

can you just implicitly adopt those... and i guess flip off `sourceType` and `globalReturn` explicitly in this ui file?

Reply:
removed `parser`.

I can't figure our how `parserOptions` would be merged, so I'll just leave that as-is.